napari/benchmarks/utils.py

```diff
@@ -1,6 +1,44 @@
+from functools import lru_cache
+
+import numpy as np
+from skimage import morphology
+
+
 class Skiper:
     def __init__(self, func):
         self.func = func
 
     def __contains__(self, item):
         return self.func(item)
+
+
+@lru_cache
+def gen_blobs(shape, dtype, blob_count=144):
```

Comment:
We definitely need a docstring, including the limitation that this is 2D or 3D only. Personally, I feel like a huge value-add would be to make this n-dimensional, for example with 3D blobs moving about in the added dimensions with brownian motion. That could come in a later PR but just thinking out loud about the kinds of limitations I find when working with skimage binary blobs.

I would also suggest a more descriptive name. At a minimum, `generate_blobs`, but, perhaps better, something like `labeled_particles`?

Also, I think everything other than shape should be a kwarg, dtype should be optional and default to the smallest dtype for the given number of blobs. I would also rename `blob_count` to just `n=`.

Reply:
Ah, I see below that the values are obtained from the dtype. That's also ok but then there should be a default dtype.

Reply:
Could you check now?